.env.example

```diff
@@ -1,3 +1,30 @@
+# Choose one of the following environment configuration: local (development), goerli (staging), mainnet (production)
+# This will automatically set other required environment variables to point to choosen configuration.
+
+# Development: point to your local ganache chain. Usit it when running Ocean stack locally with Barge.
+NODE_ENV = "development"
```

Comment:
I'd suggest that we should comment out two of these values for `NODE_ENV`, that way this file can work with the default value as soon as it's copied without the user having to come in here and commenting things out

Reply:
done